gax-java/dependencies.properties

```diff
@@ -79,9 +79,12 @@ maven.org_graalvm_sdk=org.graalvm.sdk:graal-sdk:22.3.5
 
 # Testing maven artifacts
 maven.junit_junit=junit:junit:4.13.2
-maven.org_mockito_mockito_core=org.mockito:mockito-core:2.28.2
+maven.org_mockito_mockito_core=org.mockito:mockito-core:4.11.0
+maven.org_mockito_mockito_junit_jupiter=org.mockito:mockito-junit-jupiter:4.11.0
```

Comment:
I think these changes are for compiling gax when generating libraries using `bazel build`?